src/css/responsive/desktop.styl

```diff
@@ -5,10 +5,26 @@ $header-height = 109px
 // BASIC
 
 #main-content
+  min-height: calc('100vh - ' + $header-height)
   margin-top: $header-height
 
 // ******************************************************
 
+// INDEX
+
+#header
+  &.index-header
+    top: calc('100vh - ' + ($margin-basic * 3))
```

Comment:
i was doing this because the header doesnt have any actual height under the nav items. to avoid missed clicks on page elements. but actually im glad u brought this up because theres a better way.